apps/console/src/extensions/components/logs/components/infinite-scroll-container.tsx

```diff
@@ -241,6 +288,82 @@ const InfiniteScrollContainer = (props: InfiniteScrollContainerPropsInterface):
         return propertyElements;
     };
 
+    const LogDataViewerPanel  : ReactElement = (
+
+        <MonacoEditor
+            loading={ <CircularProgress /> }
+            className="log-data-viewer"
+            width="100%"
+            height="100%"
+            language="javascript"
+            theme={ "vc" } // visual studio code light theme
+            value={ currentLog ? JSON.stringify(currentLog["data"], null, 2) : "{}" }
+            options={ {
+                automaticLayout: true,
+                readOnly: true
+            } }
+            data-componentid={ `${componentId}-data-viewer` }
+        />
+    );
+
+    const LogViewerToolbar : ReactElement = (
+
+        <Box className="log-data-viewer-toolbar-container">
+            <Toolbar variant="dense">
+                <Box>
+                    <Typography>
+                        {
+                            t("extensions:develop.monitor.logView.logDataviewer")
+                        }
+                    </Typography>
+                </Box>
+                <div className="actions">
+                    <div className="editor-fullscreen">
+                        <Tooltip
+                            title={ "Download" }
+                            data-componentid="log-data-viewer-download-tooltip"
+                        >
+                            <IconButton
+                                size="small"
+                                onClick={ exportCurrentLog }
+                            >
+                                {
+                                    <Icon name="download"/>
+                                }
```

Comment:
We don't need curly braces around jsx here.

```suggestion
                                <Icon name="download"/>
```